### PR TITLE
PS-8425 Fix mistakes in innodb_corrupt_table_action (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
M docs/reliability/innodb_corrupt_table_action.md

### CODE REVIEW

File: docs/reliability/innodb_corrupt_table_action.md [hunk 1]
```diff
@@ -45,6 +45,6 @@ This feature adds a new system variable.
 
 * With the default value, `assert`, *XtraDB* will intentionally crash the server with an assertion failure as it would normally do when detecting corrupted data in a single-table tablespace.
 
-* If the `warn` value is used it will pass corruption of the table as `corrupt table` instead of crashing itself. For this to work innodb_file_per_table should be enabled. All file I/O for the data file after detected as corrupt is disabled, except for the deletion.
+* If the `warn` value is used it will pass corruption of the table as `corrupt table` instead of crashing itself. For this to work innodb_file_per_table should be enabled. All file I/O for the data file after being detected as corrupt is disabled, except for the deletion.
```
Comment (alina-derkach-oaza) [inline docs/reliability/innodb_corrupt_table_action.md]: Add comma - If the `warn` value is used,

Comment (patrickbirch) [reply docs/reliability/innodb_corrupt_table_action.md]: Rewritten

File: docs/reliability/innodb_corrupt_table_action.md [hunk 2]
```diff
@@ -43,8 +43,8 @@ This feature adds a new system variable.
 | Default      | assert                |
 | Range        | assert, warn, salvage |
 
-* With the default value, `assert`, *XtraDB* will intentionally crash the server with an assertion failure as it would normally do when detecting corrupted data in a single-table tablespace.
+* With the default option value is `assert`, then *XtraDB* intentionally crashes the server with an assertion failure as it would normally do when detecting corrupted data in a single-table tablespace.
```
Comment (alina-derkach-oaza) [inline docs/reliability/innodb_corrupt_table_action.md]: With the default option value is `assert`, then *XtraDB* intentionally crashes   ->   If the default option value is `assert`, *XtraDB* intentionally crashes

File: docs/reliability/innodb_corrupt_table_action.md [hunk 3]
```diff
@@ -43,8 +43,8 @@ This feature adds a new system variable.
...
 
-* If the `warn` value is used it will pass corruption of the table as `corrupt table` instead of crashing itself. For this to work innodb_file_per_table should be enabled. All file I/O for the data file after detected as corrupt is disabled, except for the deletion.
+* Enabling `innodb_file_per_table` and using the `warn` option causes *XtraDB* to pass the table corruption as `corrupt table` instead of crashing the server. Detecting the file as corrupt also disables the file I/O for that data file, except for the deletion operation.
 
-* When the option value is `salvage`, *XtraDB* allows read access to a corrupted tablespace, but ignores corrupted pages”. You must enable innodb_file_per_table.
+* Enabling `innodb_file_per_table` and using the `salvage` option causes *XtraDB* to allow read access to the corrupted tablespace, but ignores any corrupted pages.
```
Comment (alina-derkach-oaza) [inline docs/reliability/innodb_corrupt_table_action.md]: Remove comma - > tablespace,

Comment (patrickbirch) [reply docs/reliability/innodb_corrupt_table_action.md]: done

File: docs/reliability/innodb_corrupt_table_action.md (hunk 2 above)
Comment (patrickbirch) [reply docs/reliability/innodb_corrupt_table_action.md]: Should be a gerund to match the other two options.

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**